tools/shaderc/shaderc.cpp

```diff
@@ -1007,6 +1007,7 @@ namespace bgfx
 			  "  -f <file path>                Input file path.\n"
 			  "  -i <include path>             Include path (for multiple paths use -i multiple times).\n"
 			  "  -o <file path>                Output file path.\n"
+			  "  -c, --console                 Output to console.\n"
```

Comment:
I would just call this `--stdout` without short form...

Reply:
Sure, that makes sense, I'll update